node/connection.js

```diff
@@ -529,14 +534,11 @@ TChannelConnection.prototype.resetAll = function resetAll(err) {
             return;
         }
 
-        var info = {
+        var info = req.extendLogInfo({
```

Comment:
We are going to store lazy requests in the operations in the future.

Lazy requests do not have a bunch of fields so we need branching between eager vs lazy requests.